### PR TITLE
Merge OneConfigLoader and OneConfigWrapper into one functional package

### PR DESCRIPTION
**Pull request by lineargraph**

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            when (requested.id.id) {
+                "gg.essential.loom" -> useModule("gg.essential:architectury-loom:${requested.version}")
+            }
+        }
+    }
+}
+
+rootProject.name = "OneConfigStages"
+
+include("Wrapper")
```
Comment (qtdyn) [inline settings.gradle.kts:23]: maybe lowercase the submodules name

**Comment by qtdyn** [comment]
okay nevermind about packages this would break everything lmao

**Comment by lineargraph** [comment]
How would this break everything?

File: settings.gradle.kts (hunk 1 above)
Comment (lineargraph) [reply settings.gradle.kts:23]: I'm changing the name below to the old names, this is just the folder name

**Comment by Wyvest** [comment]
~~> okay nevermind about packages this would break everything lmao~~

~~should be fine, no? will wait for a response but I dont think this would break things~~

nvm lmao broken English moment

**Review by nextdayy — APPROVED** [review]
seems legit, will need testing though.

**Review by pauliesnug — CHANGES_REQUESTED** [review]
lgtm, only problems is that the folder names are uppercase ew and also would it be possible for there to be a common directory since there is a lot of shared code between the loader and wrapper cc @Wyvest

**Comment by lineargraph** [comment]
yes, having a common module is the next step in my agenda. also upper case folder names are stylish modern and the next hot thing.

**Comment by pauliesnug** [comment]
the agenda !!

**Comment by lineargraph** [comment]
yes, the agenda, codename: enigma

**Comment by qtdyn** [comment]
wouldn't changing the package break existing/old loader dependencies/versions?

**Comment by lineargraph** [comment]
I try to preserve artifact coordinates: here is the output of `./gradlew publishToMavenLocal` 

```tree
/home/nea/.m2/repository/
└── cc
    └── polyfrost
        ├── oneconfig-loader-launchwrapper
        │   ├── 1.0.0-alpha10
        │   │   ├── oneconfig-loader-launchwrapper-1.0.0-alpha10.jar
        │   │   ├── oneconfig-loader-launchwrapper-1.0.0-alpha10.module
        │   │   ├── oneconfig-loader-launchwrapper-1.0.0-alpha10-obf.jar
        │   │   └── oneconfig-loader-launchwrapper-1.0.0-alpha10.pom
        │   └── maven-metadata-local.xml
        └── oneconfig-wrapper-launchwrapper
            ├── 1.0.0-alpha8
            │   ├── oneconfig-wrapper-launchwrapper-1.0.0-alpha8.jar
            │   ├── oneconfig-wrapper-launchwrapper-1.0.0-alpha8.module
            │   ├── oneconfig-wrapper-launchwrapper-1.0.0-alpha8-obf.jar
            │   └── oneconfig-wrapper-launchwrapper-1.0.0-alpha8.pom
            └── maven-metadata-local.xml
``` 

I think this matches the artifact coordinates already present in [your](https://repo.polyfrost.cc/#/releases/cc/polyfrost/oneconfig-wrapper-launchwrapper/) [repo](https://repo.polyfrost.cc/#/releases/cc/polyfrost/oneconfig-loader-launchwrapper/)

**Comment by lineargraph** [comment]
I guess i should remove the -obf artifacts

**Comment by lineargraph** [comment]
bump

**Pull request merged**